colossalai/auto_parallel/passes/runtime_apply_pass.py

```diff
@@ -45,6 +47,52 @@ def runtime_apply_for_iterable_object(node: Node, origin_dict: Dict, input_dict:
     return rst
 
 
+def construct_meta_info(node: Node, user_node: Node) -> MetaInfo:
```

Comment:
This part is to construct `MetaInfo` for the shape consistency node, I add a TODO here as there is no need to recompute this `comm_cost` as it is computed in the node handler.